blackbox/docs/appendices/release-notes/unreleased.rst

```diff
@@ -115,9 +119,6 @@ Changes
 Fixes
 =====
 
-- Fixed a regression introduced in 4.0 that broke the ``MATCH`` predicate if
```

Comment:
This entry should be kept.

Reply:
I have restored it, I would not be able to explain how it went missing, apologies.